barretenberg/cpp/src/barretenberg/stdlib/recursion/honk/verifier/goblin_verifier.test.cpp

```diff
@@ -210,7 +212,8 @@ template <typename BuilderType> class GoblinRecursiveVerifierTest : public testi
             auto composer = get_outer_composer<OuterBuilder>();
             auto instance = composer.create_prover_instance(outer_circuit);
             auto prover = composer.create_prover(instance);
-            auto verifier = composer.create_verifier(instance->verification_key);
+            auto verifier_instance = composer.create_verifier_instance(instance);
```

Comment:
In the recursive verifiers (except PG which doesnt have inner and outer), `get_outer_composer` uses a constexpr if clause to figure out the right type for the composer. Before, this was enough, as we were computing the verfication key through the composer. The most simple way to get the right type for teh verification key here is to create a `VerifierInstance` to limit the scope of this PR, as we are planning to refactor the recursive verfiers anyways.